exercises/practice/minima/.docs/instructions.md

```diff
@@ -0,0 +1,7 @@
+# Instructions
+
+Consider a polynomial function in the form f(x) = ax^3 + bx^2 + cx + d. Local
+minima are defined as f(x) < f(x - delta) and f(x) < f(x + delta). Using this
+definition, derive an expression for a local minimum in Z3 and to solve for
+a local minima on the function with the given coefficients (`a, b, c, d`).
```

Comment:
This last sentence is confusing and we also don't need to specifically mention using Z3 since this is a Z3 track. I would update to: "Using this definition, derive an expression for the local minimum and also solve for a local minima with the given coefficients ('a, b, c, d')."